api/src/resolvers/london/service-cypher.js

```diff
@@ -0,0 +1,44 @@
+export const checkFormFilledThisWeek = `
```

Comment:
I don't think we should use the words "London", "UK" in the file names. It's too specific

Reply:
Okay will refactor them everywhere